rpc/namespaces/ethereum/eth/api.go

```diff
@@ -201,7 +201,7 @@ func (e *PublicAPI) GasPrice() (*hexutil.Big, error) {
 		result *big.Int
 		err    error
 	)
-	if head := e.backend.CurrentHeader(); head.BaseFee != nil {
+	if head := e.backend.CurrentHeader(); head.BaseFee != nil && head.BaseFee.BitLen() > 0 {
```

Comment:
Ensure `eth_gasPrice` returns the minimum gas price when the fee market is disabled

Reply:
There's also the `baseFeePerGas` in block response, dynamic fee tx will use that to decide the gas prices, but since `min-gas-price` is configured per validator, we shouldn't include that in the block response. so there'll be inconsistencies here.

Reply:
> I think the situation of enabling London HF but disabling the feemarket module shouldn't happen in the first place.

I agree with this. This ideally should be checked in the `InitGenesis`